.github/workflows/release.yml

```
@@ -0,0 +1,48 @@
+name: Release
+
+on:
+  release:
+    types:
+      - created
+
+permissions:
+  contents: read
+
+jobs:
+  push:
+    if: github.repository == 'mime-types/mime-types-data'
+    runs-on: ubuntu-latest
+
+    permissions:
+      contents: write
+      id-token: write
+
+    steps:
+      # Set up
+      - uses: actions/checkout@11bd71901bbe5b1630ceea73d27597364c9af683 #v4.2.2
+        with:
+          persist-credentials: true
+
+      - id: rubygems
+        run: ruby -e 'print "version=", Gem::Specification.load(ARGV[0]).rubygems_version, "\n"' mime-types-data.gemspec >> $GITHUB_OUTPUT
+
+      - name: Set up Ruby
+        uses: ruby/setup-ruby@d781c1b4ed31764801bfae177617bb0446f5ef8d #v1.218.0
+        with:
+          bundler-cache: false
+          ruby-version: ruby
+
+      - name: Install dependencies
+        run: |
+          gem update --system=${RUBYGEMS_VERSION}
+          bundle install --jobs 4 --retry 3
+        env:
+          RUBYGEMS_VERSION: ${{ steps.rubygems.outputs.version }}
+
+      - name: Set VERSION
+        run: printf "VERSION=%s\n" "${TAG_NAME#v}" >> $GITHUB_ENV
```

```suggestion
        run: printf "VERSION=%s\n" "${TAG_NAME#v}" >> "$GITHUB_ENV"
```

I'll need to think about this part a bit, because for _this_ gem, I want to make it so that this can be completely automated for the data releases (and because all other PRs that get added are reviewed and added prior to each release, I don't mind the data releases being *completely* automated to where they may not even open a PR anymore…eventually).